### PR TITLE
chore: remove dckc from * "everything" owner list

### PR DESCRIPTION
**Pull request by dckc**
I'm assigned as an owner of #4962 , but I don't see much reason for that. I think the `*` line at the top was intended to match things that don't match other lines, but I don't think it works that way. I think it tags me as an owner of every PR.

refs: #3382

### CODE REVIEW

**Review by michaelfig — APPROVED** [review]

**Comment by michaelfig** [comment]
> I'm assigned as an owner of #4962 , but I don't see much reason for that.

That PR changes `packages/vat-data`, which has no code owners by default, so you were tagged.

> I think the `*` line at the top was intended to match things that don't match other lines, but I don't think it works that way. I think it tags me as an owner of every PR.

I can vouch that it doesn't tag you for everything.  Is the right thing to do instead to add a line for `/packages/vat-data` ?

**Comment by dckc** [comment]
> I can vouch that it doesn't tag you for everything.

The fact that you knew that and I didn't tells me it's best that we merge this PR. Thanks for approving.

> Is the right thing to do instead to add a line for `/packages/vat-data` ?

Not instead but perhaps in addition. I leave that to you / others.

**Pull request merged**
